htdocs/install/mysql/migration/3.7.0-3.8.0.sql

```diff
@@ -797,6 +797,7 @@ ALTER TABLE llx_societe_remise_except MODIFY COLUMN description text NOT NULL;
 update llx_opensurvey_sondage set format = 'D' where format = 'D+';
 update llx_opensurvey_sondage set format = 'A' where format = 'A+';
 
+ALTER TABLE llx_propal_merge_pdf_product ADD COLUMN lang varchar(5) DEFAULT NULL;
```

Comment:
No structure change are allowed into a stable version.
This seems to be required to fix a hidden feature. So we must just consider the feature as not ready into 3.8 and fix it into dev.